migrations/20161123212652_add-claim-column-payment-status.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.table('Claim', function (table) {
+    table.string('PaymentStatus', 20).notNullable().defaultTo('PENDING')
```

Comment:
may need to make this nullable, knex has problems applying non-nullable columns to existing data. Also I think default value 'PENDING' could be misleading as it implies claim payment is in progress